content/13.conclusion.md

```diff
@@ -1 +1,7 @@
 ## Conclusion {#conclusion}
+
+Deep learning techniques have the potential for wide use in biology, meeting or exceeding the performance of both humans and the current state-of-the art algorithms in a variety of tasks.
+Beyond simply achieving performance, deep learning has the potential to generate novel biological insights that may then be applied back to fundamental research.
+To realize this potential, the use of deep learning as a research tool must be approached as any other tool would be: scientifically and thoughtfully.
```

Comment:
Nothing wrong with it, but we could simplify this to

> To realize this potential, the use of deep learning as a research tool must be approached scientifically and thoughtfully.

Reply:
While simpler, I think that the proposed alternative does not capture the key phrase "any other tool". However, the current line 5 might not get what I am trying to say across either.

To make what I was trying to get at concrete, most biologists don't anthropomorphize PCR or see it as black magic; they simply use it as a powerful but delicate tool. In an ideal world, I'd like to see DL used similarly to PCR in that respect.

Reply:
I see. Maybe adding a concrete example would help in that case then? I was more thinking about tools as "programming" or things like that (in that context, using it properly in terms of documenting, making sure the outcome is understandable and reproducible etc.). Regarding PCR, your comment makes sense then, but since the audience will be mostly biologists, maybe a concrete example (like the PCR one) that everyone can relate to would be nice? (Esp. because I think our article is very vague in general ... readers might appreciate some examples)